### DISCUSSION TITLE
Can't copy default config to edit

### DISCUSSION THREAD

**Discussion by Dyltong** [abraunegg/onedrive#1852] [Q&A] [answered]
Hi all. I am trying to copy the default config file so I can change the sync directory (sync_dir).

I try the following from the configuration guide, but it says no such directory exists (when copying the config). 
```
mkdir -p ~/.config/onedrive
cp ./config ~/.config/onedrive/config
nano ~/.config/onedrive/config

```
a) Is there something I need to change?
b) is there another way of changing the sync_dir, I want it to be /hdd

**Comment by abraunegg** [comment, marked as answer]
@Dyltong 
As per the documentation, this is *just* an example .... 

By default, in your `~/.config/onedrive/` location - if you are using the application defaults, there will be no configuration file in that location.

Additionally, those instructions are (were) written based on installing from source / being in the cloned github directory (hence the `cp ./....`) which really should be updated / cleaned up and made more relevant.

To create a 'config' file you can:
1. Get the config file from this location using wget: https://raw.githubusercontent.com/abraunegg/onedrive/master/config
2. Create a new file called `config` and add your `sync_dir =` modified item to that

**Reply by Dyltong** [reply]
Many thanks @abraunegg, I'll give it a go. 

I appreciate the work you are doing on this.

**Comment by Dyltong** [comment]
Thanks for the response above @abraunegg

What worked for me in Ubuntu 20.04 is this:

```
mkdir -p ~/.config/onedrive
sudo wget  https://raw.githubusercontent.com/abraunegg/onedrive/master/config -P  ~/.config/onedrive/ # download the default file
sudo nano ~/.config/onedrive/config #to edit the config

```
